components/Dialogs/examples/DialogsAlertCustomizationExampleViewController.swift

```diff
@@ -216,6 +220,32 @@ class DialogsAlertCustomizationExampleViewController: MDCCollectionViewControlle
     return alert
   }
 
+  func performDarkThemed() -> MDCAlertController {
+    let alert = MDCAlertController(title: "Dark Theme",
+                                   message: "Lorem ipsum dolor sit amet, consectetur adipiscing")
+    alert.titleIcon = sampleIcon()
+
+    alert.addAction(MDCAlertAction(title:"All Right", emphasis: .high, handler: handler))
+    alert.addAction(MDCAlertAction(title:"Not Now", emphasis: .medium, handler: handler))
+    alert.addAction(MDCAlertAction(title:"Later", emphasis: .low, handler: handler))
+
+    let darkScheme: MDCContainerScheme = {
```

Comment:
I couldn't find any example (in the MDC catalog) that uses dark schemes. 
I'm wondering if it makes more sense to add a default dark scheme (ie: MDCColorSchemeDefaultsMaterialDark201811).

Reply:
ok then we can do that in parallel (do block this work) Filling an issue might make it happen faster.

Reply:
Dark theming is not resolved on MDC. I added a reference to the issue that fixes dark theming in GM, but for MDC there is more work that needs to happen for dark theming to work (described in the issue). 
The approach taken in this PR is to write new themers (theming extensions in this case) - since that is what we recommend clients should do when the existing themers are not enough.